docs/DeveloperGuide.adoc

```diff
@@ -521,11 +521,11 @@ The calendar mechanism is facilitated by the `DatePickerSkin` skin instance whic
 
 Suppose the address book is modified (e.g. using `add`, `edit`, `addtask`, `deletetask`).
 
-After the modification of the address book i.e. saving of data to the `addressbook.xml` file, `ObservableList` of `Person` and `Task` classes stored in the address book are passed into the `CalendarPanel` Ui through `Model`.
+After the modification of Mobilize's address book (i.e. saving of data to the `addressbook.xml` file), `ObservableList` of `Person` and `Task` classes stored in the address book are passed into the `CalendarPanel` Ui through `Model`.
 
 Next, `getDayCellFactory` will format the `DateCell` of the `DatePicker`. The formatting is done with `Birthday` and `Deadline` as conditions and colour and tooltip of `DateCell` as variables to be change by the formatter.
 
-By selecting the date on the calendar, the date value from the `DatePicker` will be captured and be parsed as a string. The string is then pass to `Logic` to execute the `FindCommand` which will show all persons and tasks having `Birthday` and `Deadline` on that specific date.
+By selecting the date on the calendar, the date value from the `DatePicker` will be captured and be parsed as a string. The string is then passed to `Logic` to execute the `FindCommand` which will show all contacts and tasks having `Birthday` and `Deadline` on that specific date.
```

Comment:
I suggest you change birthday and deadline to just birthday and deadline without the highlighting. Because you're not referring to the birthday and deadline classes over here.